packages/create-catalyst/jest.config.cjs

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  extensionsToTreatAsEsm: ['.ts'],
+  transform: {
+    '^.+\\.(t|j)s?$': '@swc/jest',
+  },
+  transformIgnorePatterns: [],
+  // moduleFileExtensions: ['ts', 'tsx', 'js', 'jsx', 'json', 'node'],
+  moduleNameMapper: {
+    '^(\\.{1,2}/.*)\\.js$': '$1',
+  },
+  testEnvironment: 'node',
+};
```

Comment:
All you need is this (see `tsconfig.json` comments):
```suggestion
module.exports = {
  transform: {
    '^.+\\.(t|j)s?$': '@swc/jest',
  },
};
```

Reply:
All of this looks good! However, I do need to keep the `moduleNameMapper` because this is a `"type": "module"` package, and the `.js` extensions on imports are not found if the mapper is not present

Reply:
Oops 🤦‍♂️ Had to restart ESLint, and now the changes are working. Removing moduleNameMapper! Thank you!